dimmer.el

```diff
@@ -114,18 +114,27 @@
   :group 'dimmer)
 
 (make-obsolete-variable
- 'dimmer-exclusion-regexp
```

Comment:
At this point `dimmer-exclusion-regexp` should continue to be marked obsolete because the semantics of the variable were completely different than the new regexp lists.  We should keep this code. Also, I'm pretty sure the `define-obsolete-variable-alias` macro handles marking the old variable obsolete, so it's not necessary to use `(make-obsolete-variable 'dimmer-exclustion-regexp-list ...)` if we're also providing the alias on lines 122-123.